sktime/transformations/bootstrap/_tsbootstrap.py

```diff
@@ -0,0 +1,135 @@
+        Whether to include the actual data in the output.
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.transformations.bootstrap import TSBootstrapAdapter
+    >>> from tsbootstrap import (
+    ...    MovingBlockBootstrap,
+    ...    MovingBlockBootstrapConfig
+    ... ) # doctest: +SKIP
+    >>> y = load_airline()
+    >>> config = MovingBlockBootstrapConfig(10, n_bootstraps=10)  # doctest: +SKIP
+    >>> bootstrap = TSBootstrapAdapter(MovingBlockBootstrap(config))  # doctest: +SKIP
+    >>> result = bootstrap.fit_transform(y)  # doctest: +SKIP
+    """
+
+    _tags = {
+        "scitype:transform-input": "Series",
+        "scitype:transform-output": "Panel",
+        "scitype:instancewise": True,  # is this an instance-wise transform?
+        "X_inner_mtype": "pd.DataFrame",  # which mtypes do _fit/_predict support for X?
+        "y_inner_mtype": "None",  # which mtypes do _fit/_predict support for y?
+        "capability:inverse_transform": False,
+        "skip-inverse-transform": True,  # is inverse-transform skipped when called?
+        "univariate-only": False,  # can the transformer handle multivariate X?
+        "handles-missing-data": True,  # can estimator handle missing data?
+        "X-y-must-have-same-index": False,  # can estimator handle different X/y index?
+        "enforce_index_type": None,  # index type that needs to be enforced in X/y
+        "fit_is_empty": True,  # is fit empty and can be skipped? Yes = True
+        "transform-returns-same-time-index": False,
+        "python_dependencies": ["tsbootstrap>=0.0.3"],
+    }
+
+    def __init__(
+        self,
+        bootstrap,
+        include_actual=False,
+    ):
+        self.bootstrap = bootstrap
+        self.include_actual = include_actual
+        super().__init__()
+
+    def _transform(self, X, y=None):
+        """Transform X and return a transformed version.
+
+        private _transform containing core logic, called from transform
+
+        Parameters
+        ----------
+        X : Series or Panel of mtype X_inner_mtype
+            if X_inner_mtype is list, _transform must support all types in it
+            Data to be transformed
+        y : Series or Panel of mtype y_inner_mtype, default=None
+            Additional data, e.g., labels for transformation
+
+        Returns
+        -------
+        transformed version of X
+        """
+        bootstrapped_samples = self.bootstrap.bootstrap(X, test_ratio=0)
+
+        bootstrapped_samples = [pd.DataFrame(sample) for sample in bootstrapped_samples]
+
+        boostrapped_df = pd.concat(
+            bootstrapped_samples,
+            keys=[f"synthetic_{i}" for i in range(len(bootstrapped_samples))],
+        )
+
+        _X = X.copy()
+        if self.include_actual:
+            _X.index = pd.MultiIndex.from_product([["actual"], range(len(X))])
+            return pd.concat([_X, boostrapped_df], axis=0)
+        else:
```

Comment:
Since if has a return, else is probably unnecessary and the next return can be de-indented.

Non-blocking, of course.

Reply:
You are right, but for me the current version is easier to read :)